cmd/zc_traverser_local.go

```diff
@@ -463,21 +464,27 @@ func (t *localTraverser) Traverse(preprocessor objectMorpher, processor objectPr
 				if t.incrementEnumerationCounter != nil {
 					t.incrementEnumerationCounter(entityType)
 				}
+				ctime := fileInfo.Sys().(*syscall.Stat_t).Ctim
```

Comment:
Sys() may be null on Windows and/or may not be of type *syscall.Stat_t, so it doesn't look safe to do this in common code.

Reply:
I changed it to know the performance number. Intention is to use GetExtendedProperties which will be platform dependent code. Reverted back to original